Dockerfile-hydroshare

```diff
@@ -3,7 +3,7 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
-RUN pip install markdown
+RUN pip install markdown nose-timer
 
```

Comment:
Once is for new dev setup another for the normal build

Reply:
Fixed in 8df0ec4e6

Reply:
@cuahsimarko You will have to rebuild a base image on dockerhub and then pointing hydroshare to the updated base image if the tag is changed. @sblack-usu can advise if needed.